src/CodeFormatter/Program.cs

```diff
@@ -111,17 +152,32 @@ private static int Main(string[] args)
 
             try
             {
-                RunAsync(
-                    projectOrSolutionPath,
-                    ruleTypeBuilder.ToImmutableArray(),
-                    fileNamesBuilder.ToImmutableArray(),
-                    configBuilder.ToImmutableArray(),
-                    copyrightHeader,
-                    allowTables,
-                    convertUnicode,
-                    verbose,
-                    ct).Wait(ct);
-                Console.WriteLine("Completed formatting.");
+                var rules = ruleTypeBuilder.ToImmutableArray();
+                var fileNames = fileNamesBuilder.ToImmutableArray();
+                var config = configBuilder.ToImmutableArray();
+
+                foreach (string projectOrSolution in projectOrSolutionPath)
+                {
+                    RunAsync(
+                        projectOrSolution,
+                        rules,
+                        fileNames,
+                        config,
+                        copyrightHeader,
+                        allowTables,
+                        convertUnicode,
+                        verbose,
+                        ct).Wait(ct);
+                }
+
+                if (projectOrSolutionPath.Length > 1)
+                {
+                    Console.WriteLine("Completed formatting {0} projects/solutions.", projectOrSolutionPath.Length);
+                }
+                else
+                {
+                    Console.WriteLine("Completed formatting.");
+                }
```

Comment:
User already gets feedback for every project / solution that is formatted. The output looks like this:
--------------------
Proj1.csproj
        Syntax Pass
        Local Semantic Pass
        Global Semantic Pass
Total time 00:00:00.0798914
Proj2.csproj
        Syntax Pass
        Local Semantic Pass
        Global Semantic Pass
Total time 00:00:00.0225966
Completed formatting 2 projects/solutions.